tinymce-angular-component/src/main/ts/editor/editor.component.ts

```diff
@@ -176,7 +176,9 @@ constructor(
     if (typeof this.initialValue === 'string') {
       this.ngZone.run(() => {
         editor.setContent(this.initialValue);
-        this.onChangeCallback(editor.getContent({ format: this.outputFormat }));
+        if (editor.getContent() !== this.initialValue) {
```

Comment:
Why is it `initialValue` and not the current value? Or is `initialValue` updated?

Reply:
I think this would be when you for example got something like `FormControl('<p>a</p><p>b</p>');` with tinymce-angular which the editor would return as `<p>a</p>↲<p>b</p>` and thus putting the editor out of sync with the model if it weren't for the onChangeCallback below.